test/react-bem.test.js

```diff
@@ -3,20 +3,32 @@ describe("Completer Integration Tests", function() {
   var TestUtils = React.addons.TestUtils;
 
   var connect = function(Component) {
-    return TestUtils.renderIntoDocument(Component());
+    return TestUtils.renderIntoDocument(React.createElement(Component));
   };
 
   var unmount_component = function(component) {
-    React.unmountComponentAtNode(component.getDOMNode().parent);
+    React.unmountComponentAtNode(component.getDOMNode().parentNode);
   };
 
+  var getTypeOfDomNode = function (el) {
+    var reactComponent = el._reactInternalComponent._currentElement;
+    var name = '';
+    if (typeof reactComponent.type === "string") {
+      name = reactComponent.type
+    } else {
+      name = reactComponent.type.displayName
+    }
+
+    return name;
+  }
```

Comment:
No worries. This is great! Sorry for the delay. I'll accept the pull request soon. Really appreciate the  hard work [=